include/deal.II/base/cuda_size.h

```diff
@@ -33,6 +33,11 @@ namespace CUDAWrappers
    * changed depending on the architecture the code is running on.
    */
   constexpr int chunk_size = 1;
+
+  /**
+   * Define the number of threads in a warp.
+   */
+  constexpr int warp_size = 32;
```

Comment:
Can we move this to the cmake setup? We could configure it with some flags like `--D DEAL_II_WITH_AMD`...

Reply:
That's the plan but first I want to have everything working on AMD GPU and then I'll add the options in CMake.